i18n.ts

```diff
@@ -0,0 +1,18 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import Backend from 'i18next-http-backend';
+
+i18n
+  .use(Backend)
+  .use(initReactI18next)
+  // init i18next
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init({
+    fallbackLng: "en",
+    debug: true,
+    // resources: {}, // Where we can put translations' files, not needed if using react Backend
```

Comment:
Didn't have to use it as the i18next-http-backend defaults works nicely. 
Using it like in apps/ledger-live-desktop/src/renderer/i18n/init.ts is still feasible though